app/forms/field_handlers/number_handler.py

```diff
@@ -87,7 +87,7 @@ def get_field_references(
     def _get_number_field_validators(
         self,
     ) -> list[Union[NumberCheck, NumberRange, DecimalPlaces]]:
-        answer_errors = self.error_messages.copy()
+        answer_errors = dict(self.error_messages).copy()
```

Comment:
You don't need the `.copy()`, calling `dict()` on a mapping will return a copy already.